administration/css.md

```diff
@@ -30,21 +34,37 @@ For more strategic and programs-level conversations.
 
 E-mail address: **`fsp@codeforscience.org`**.
 
+## CS&S HR
+
+For confidential Human Resources needs.
+
+E-mail address: **`hr@codeforscience.org`**.
+
+## CS&S Bills
+
+For sending invoices. This inbox is automated and not monitored.
+
+E-mail address: **`bills@codeforscience.org`**.
+
 ## Google Group
 
 We have a dedicated Google Group and e-mail that we use to provide CS&S team members access to our files.
 
 - **`css-admin@2i2c.org`**
 
 This is because they need visibility into many of our operational information for bookkeeping and accountability purposes.
-It contains several adminsitrative and organizational leaders of CS&S.
+It contains several administrative and organizational leaders of CS&S.
 They are all listed as managers, so can add people on their own.
 
 In Google Drive, you may add the `css-admin@2i2c.org` group to the sharing list for a folder or file and they will have access.
 
 ## Asana board
 
-CS&S has an Asana board that they occasionally use to tack work items.
+CS&S has an Asana board that they occasionally use to track work items.
 You can find it at this link:
 
 [CS&S Asana Board](https://app.asana.com/0/1200569652722016/list).
+
+```{note}
+The list above is fairly comprehensive – there is no need to use any other aliases (such as `accouting@codeforscience`, etc.) since they are intended for outgoing use.
```

Comment:
```suggestion
The list above is fairly comprehensive – there is no need to use any other aliases (such as `accounting@codeforscience`, etc.) since they are intended for outgoing use.
```